pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -44,39 +39,51 @@
 
 /**
  *
+ * This class is not thread safe.
```

Comment:
non of the operators as threadsafe. each task should create their own operator instance.